docs/source/deploying-volttron/multi-platform/multi-platform-rabbitmq-deployment.rst

```diff
@@ -261,10 +265,13 @@ upstream servers on the downstream server and make the VOLTTRON exchange
 .. note::
 
     These commands only remove the federation parameter from RabbitMQ and certificate entries from rabbitmq_federation_config.yml on the publisher node.
-    `It does not remove the actual certificates.` If you need to rerun the federation command again for the same setup and need to create fresh certificates, then you will
-    need to manually remove public and private certificates. Private certificates will be in
+    `It does not remove the actual certificates.` If you need to rerun the federation command again for the same setup
```

Comment:
Rerunning the federation comman with the same setup creates a fresh set of certifiicates.